x-pack/plugins/rule_registry/server/alert_data_client/audit_events.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EcsEventOutcome, EcsEventType } from 'src/core/server';
+import { AuditEvent } from '../../../security/server';
+
+export enum AlertAuditAction {
+  GET = 'alert_get',
+  UPDATE = 'alert_update',
+  FIND = 'alert_find',
+}
```

Comment:
Currently, the alerting framework audit events are all prefixed with `alert_`. We are happy to rename, but I believe the audit actions are meant to include the saved object type, which in our case is still `alert`. @legrego can you confirm that this is the case (that the audit event should be prefixed with the SO type)? 

We are looking into trying to migrate to a SO type that follows the new terminology (`rule`) but if it's possible, it would be done for 8.0. Not sure what the right solution is here, but having audit events for both rules and alerts that use `alert_*` will likely lead to confusion.

Reply:
We generally try to keep these names in sync, but I think what's more important is that the audit events make sense to consumers of the audit log - they may or may not be familiar with how saved objects are represented internally. I think having the audit events named after the user-facing entities makes the most sense.

> having audit events for both rules and alerts that use alert_* will likely lead to confusion.

I agree, they should have distinct audit events since they are in fact distinct entities that are authorized independently

Reply:
@ymao1 I wonder if we should audit (no pun intended) the plugins a bit to make sure we're being consistent with naming. It seems parts fo the alerting plugin have moved to using `alerting` and some still use `alerts` and on our end we sometimes use `rac` and sometimes `alerts`. Imho what would make sense for the user is the use of `rules` and `alerts`, even `alerting` and `alerts` may be confusing.

Reply:
Created this issue for updating alerting framework's audit events: https://github.com/elastic/kibana/issues/101476